### ISSUE TITLE
Complicated alias breaks git-up

### ISSUE THREAD

**Issue by mcepl** [msiemens/PyGitUp#42]
@whot published on [his blog](http://who-t.blogspot.cz/2012/06/git-branch-info.html) this absolutely distastefully complicated git alias:
```sh
branch-info = "!sh -c 'git branch --list --no-color | \
    sed -e \"s/*/ /\" | \
    while read branch; do \
    git log -1 --format=format:\"%Cred$branch:%Cblue %s %Cgreen%h%Creset (%ar)\" $branch; \
    done'"
```
Yes, it is a way past the moment when it should be split into a separate shell script (and actually Peter apparently got it on his own, and it is now https://github.com/whot/git-branch-tools/blob/master/git-branch-description (if I am not mistaken)), but it still works. However, when I add it to my ``~/.gitconfig`` ``git-up`` breaks with this traceback:
```
Traceback (most recent call last):
  File "/home/matej/.local/bin/git-up", line 9, in <module>
    load_entry_point('git-up==1.2.1', 'console_scripts', 'git-up')()
  File "/home/matej/.local/bin/gitup.py", line 572, in run
    gitup = GitUp()
  File "/home/matej/.local/bin/gitup.py", line 135, in __init__
    if not any(b.tracking_branch() for b in self.repo.branches):
  File "/home/matej/.local/bin/gitup.py", line 135, in <genexpr>
    if not any(b.tracking_branch() for b in self.repo.branches):
  File "/home/matej/.local/lib/python2.7/site-packages/git/refs/head.py", line 160, in tracking_branch
    if reader.has_option(self.k_config_remote) and reader.has_option(self.k_config_remote_ref):
  File "/home/matej/.local/lib/python2.7/site-packages/git/config.py", line 91, in <lambda>
    return lambda *args, **kwargs: self._call_config(attr, *args, **kwargs)
  File "/home/matej/.local/lib/python2.7/site-packages/git/config.py", line 97, in _call_config
    return getattr(self._config, method)(self._section_name, *args, **kwargs)
  File "/home/matej/.local/lib/python2.7/site-packages/git/config.py", line 54, in assure_data_present
    self.read()
  File "/home/matej/.local/lib/python2.7/site-packages/git/config.py", line 294, in read
    self._read(fp, fp.name)
  File "/home/matej/.local/lib/python2.7/site-packages/git/config.py", line 266, in _read
    raise e
ConfigParser.ParsingError: File contains parsing errors: /home/matej/.gitconfig
	[line 27]: '            sed -e \\"s/*/ /\\" | \\\n'
	[line 28]: '            while read branch; do \\\n'
	[line 30]: '            done\'"\n'
matej@mitmanek: m2crypto (master)$
```

**Comment by mcepl** [comment]
Apparently this got fixed somewhere on the way to v1.2.1

**Comment by whot** [comment]
in my defense, I haven't used that alias for years :)
